admin/lib/validator/settings-validator.js

```diff
@@ -18,6 +18,13 @@ const settingsValidationSchema = {
       options: [{ min: 1, max: 5 }] // Max number to be confirmed
     },
     errorMessage: settingsErrorMessages.loadingTimeLimit
+  },
```

Comment:
This is probably the only place along with the old HDF validator where we are still following this validation schema pattern which seems very outdated.
Why not rewrite it in the same approach that the other validators for consistency?
I'd expect it to be simple and quite cleaner.
@GuyHarwood @jon-shipley thoughts on that?

Reply:
@GeorgeChatzigiannis @GuyHarwood : It needs to be re-written and express-validator removed.   There are tickets for the remaining legacy validation to do just this.

Reply:
@danwilliams as per discussion - please switch to new validator